src/status_im/navigation/screens.cljs

```diff
@@ -136,544 +137,922 @@
     [status-im.navigation.options :as options]
     [status-im.navigation.transitions :as transitions]))
 
+(defn chat-screens
+  []
+  [{:name      :start-a-new-chat
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.new-chat}}
+    :options   {:sheet? true}
+    :component new-chat/view}
+
+   {:name      :chat
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.chat}}
+    :options   {:popGesture false
+                :animations transitions/stack-transition-from-bottom}
+    :component chat/chat}
+
+   {:name      :group-create
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.new-group}}
+    :options   {:sheet?           true
+                :skip-background? true}
+    :component group-create/view}
+
+   {:name      :group-details
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.group-details}}
+    :component group-details/group-details}
+
+   {:name      :group-add-manage-members
+    :metrics   {:flow   :messenger
+                :track? :true
+                :event  {:id :messenger.group-manage-members}}
+    :options   {:sheet? true}
+    :component group-details/add-manage-members}])
+
+(defn community-screens
+  []
+  [{:name      :discover-communities
+    :metrics   {:flow   :community
+                :track? :true
+                :event  {:id :community.discover}}
+    :component communities.discover/view}
+
+   {:name      :community-overview
+    :metrics   {:flow   :community
+                :track? :true
+                :event  {:id :community.overview}}
```

Comment:
Is the event id for finding the event in the dashboard? Can we just use the name of the screen, just to avoid creating many "slightly different" identifiers?